openslides/users/static/js/users/site.js

```diff
@@ -458,6 +475,62 @@ angular.module('OpenSlidesApp.users.site', [
             }
             $scope.sortColumn = column;
         };
+        // define custom search filter string
+        $scope.getFilterString = function (user) {
```

Comment:
This function looks a little bit like the one in motion list controller. Can we have a generic solution here?